Polyhedron/demo/Polyhedron/Plugins/IO/Polylines_io_plugin.cpp

```diff
@@ -62,7 +62,7 @@ class Polyhedron_demo_polylines_io_plugin :
 
     }
   QString name() const override{ return "polylines_io_plugin"; }
-  QString nameFilters() const override{ return "Polylines files (*.polylines.txt *.cgal)"; }
```

Comment:
Oh! That's why on my machine the files were saved with funny names! On my (Linux) machine, I have several files named like `DF.polylines.txt *.cgal`.